src/test/java/uk/gov/hmcts/reform/et/syaapi/helper/TseApplicationHelperTest.java

```diff
@@ -71,6 +71,9 @@ void applicationStatusWaitingForTheTribunal() {
             Assertions.assertEquals("Response to Amend Response",
                 caseData.getDocumentCollection().get(0).getValue().getShortDescription()
             );
+            Assertions.assertEquals("Amend Response",
```

Comment:
Amend Response is not a valid application type - I know it's only a lower case 'r' but what good is tests when the data we are using isn't even correct

```suggestion
            Assertions.assertEquals("Amend response",
```

Reply:
Test data has been updated